tests/test_basic.py

```diff
@@ -1367,6 +1367,23 @@ def test_ohmic_inputresistance():
         stimulus_current)
 
 
+def test_ohmic_inputresistance_zero_stimulus_current():
+    """Test the edge case of having 0.0 stimulus current."""
+    import efel
+    efel.reset()
+
+    time, voltage = load_ascii_input(meanfrequency1_url)
+    trace = {'T': time, 'V': voltage, 'stim_start': [-2], 'stim_end': [-1]}
```

Comment:
`ohmic_input_resistance` depends on `voltage_deflection`, which is actually very sensitive on `stim_start `and `stim_end` being correct. Could you give the actual `stim_start` and `stim_end` values here?

Reply:
Updated

Reply:
It is now the following:

`trace = {'T': time, 'V': voltage, 'stim_start': [500], 'stim_end': [900]}`